SocialFilter/sections/css_extended.txt

```diff
@@ -155,7 +155,7 @@ washingtonpost.com#?#div[data-qa="article-actions"] > div[class^="wpds-c-"] > di
 adme.media#?#article > div[class] > div[class]:has(> div[class]:contains(Поделиться этой статьёй))
 forbes.kz#?#.watches > span:contains(Поделиться этой публикацией)
 quantamagazine.org#?#.post__sidebar__content > div.post__sidebar__content__inner:has(> h3:contains(Share this article))
-mechacomic.jp#?##main > section.c-content > div.area_btn_sns:upward(1)
+mechacomic.jp#?##main section.c-content > div.area_btn_sns:upward(1)
```

Comment:
Is `main` required?

Reply:
`main` was included in the original rule.
so I followed.